src/components/select/select-list/select-list.spec.js

```diff
@@ -642,6 +642,34 @@ describe("SelectList", () => {
           { height: "150px" },
           wrapper.find(StyledSelectListContainer)
         );
+
+        jest
+          .spyOn(listElement, "clientHeight", "get")
+          .mockImplementation(() => 250);
+        jest
+          .spyOn(listActionButtonElement, "clientHeight", "get")
```

Comment:
We also don't need/want a list action button for this test

Reply:
It's gone 👋🏻